apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -81,21 +85,29 @@ const configuredUiItems = new Map<string, UiItem>([
         await MeasureTools.startup();
         MeasurementActionToolbar.setDefaultActionProvider();
       },
-      createUiItemsProvider: () => new MeasureToolsUiItemsProvider(),
+      createUiItemsProviders: () => [new MeasureToolsUiItemsProvider()],
     }
   ],
   [
     "map-layers",
     {
-      initialize: async () => MapLayersUI.initialize(),
-      createUiItemsProvider: () => new MapLayersUiItemsProvider(),
+      initialize: async () => {
+        await FrontendDevTools.initialize();
```

Comment:
as mentioned in the other comment, just pull this into the viewer's onIModelAppInit